src/frontend/src/common/queries/groups.ts

```diff
@@ -77,6 +77,12 @@ export const mapGroupMemberData = (obj: RawGroupMemberRequest): GroupMember => {
   };
 };
 
+/**
+ * starting all group-related queries with "group" allows us to invalidate
+ * all of the queries in a single call to queryClient.invalidateQueries(["group"])
+ */
+export const GROUP_QUERY_PREFIX = "group";
```

Comment:
Is this not case-sensitive? All the group queries I see start with `GROUP` for the value, eg
```
  GROUP_INVITATION_INFO = "GROUP_INVITATION_INFO",
```
... oh, wait, I see that `id` is `groupInvitationInfo`. It's the `id` that matters for invalidation, not `entities` then? Maybe call it `GROUP_QUERY_ID_PREFIX`? I dunno, maybe that's silly and would be more obvious if I knew react-query better.

Reply:
Regardless, I think if we're depending on `GROUP_QUERY_PREFIX` matching other values, it's better if we can force that in code rather than by convention. So we could change stuff to
```
id: `${GROUP_QUERY_PREFIX}InvitationInfo`,
```

Reply:
renaming to be as clear as possible seems good.  I doubt we'll ever have "react-query experts" 😅